src/reducers/index.js

```diff
@@ -10,6 +11,21 @@ import twreporterRedux from 'twreporter-redux'
 
 const { reducers, reduxStateFields } = twreporterRedux
 
+const registrationInitialState = {
+  apiUrl: '',
+  signUp: '',
+  signIn: '',
+  activate: '',
+  oAuthProviders: {
+    google: '',
+    facebook: ''
+  },
+  location: 'http://testtest.twreporter.org:3000',
+  domain: 'twreporter.org'
```

Comment:
How to configure it ?
If you want different values in different environments.

Reply:
What do mean by the word environment ? You mean without redux ? 

In the case, I configure it in server.js file

Reply:
So I update the config file. The config file contains the registration default value and the value set are created based on different env (dev/production).